_contentTemplates/common/get-started.md

```diff
@@ -13,59 +13,58 @@ The latest version of Telerik UI for Blazor is `{{site.uiForBlazorLatestVersion}
 
-If you use VS Code, you can also use our [VS Code Extension to create a Telerik-enabled project]({%slug getting-started-vs-code-integration-overview%}).
+>tip If you prefer VS Code, you can use the [VS Code Extension]({%slug getting-started-vs-code-integration-overview%}) to create a Telerik-enabled project.
 
-The rest of this article will explain the manual steps if you want to have a better understanding of the underlying process.
+If you choose to create a project with the Telerik VS Extensions, you can jump directly to [Step 3 - Add a Telerik Component to a View](#step-3---add-a-telerik-component-to-a-view). The following sections in this article explain the manual steps to configure the project so that you can better understand the underlying process.
 
-### Create a Project with the CLI
+### Create a Project with Visual Studio
 
-The next section shows how to create the Blazor through the Visual Studio interface. If you are not running Visual Studio, you can create the Blazor project from the command prompt - see the `dotnet new` command and the arguments related to Blazor apps: 
+To create a project manually, without using the Telerik VS Extensions, follow these steps:
 
-* <a href="https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new#arguments" target="_blank">MSDN: dotnet new arguments</a>
+1. Open Visual Studio 2019.
 
-The rest of this article shows some steps through the Visual Studio interface, as it is the most common IDE used with Blazor. Where such steps exist, there are links and instructions on performing them yourself (e.g., adding a nuget feed through the CLI).
+1. Create a New Project.
 
+1. Choose **Blazor App** and click **Next**. Then, choose a name and location for the project and click **Create**.
 
-### Create a Project with Visual Studio
+    ![Create new ASP.NET Core Web Application](images/create-new-application.png)
 
-To create a project manually, follow these steps:
+#end
 
-1. Open Visual Studio 2019
 
-1. Create a New Project
+#project-creation-cli
+### Create a Project with the CLI
 
-1. Choose `Blazor App` and click `Next`. Then, choose a name and location for the project and click `Create`.
+If you are not running Visual Studio, you can create the Blazor project from the command prompt. See the <a href="https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new#arguments" target="_blank">`dotnet new` command and the arguments for Blazor apps</a>.
 
-    ![Create new ASP.NET Core Web Application](images/create-new-application.png)
 #end
 
 #start-trial
-If you don't have an active license, [start a UI for Blazor trial](https://www.telerik.com/download-trial-file/v2-b/ui-for-blazor). The file this will let you download and install the components, and will also let you setup our [online NuGet feed]({%slug installation/nuget%}) automatically - make sure to select the "Set up Telerik NuGet package source" checkbox: 
```

Comment:
This is repeating the information in step 0 - download the free trial. Do we need it?

Reply:
Yes, it takes care of customer questions and tickets and avoids confusion.